pkg/controller/linstorcsidriver/linstorcsidriver_controller.go

```diff
@@ -1011,7 +1011,7 @@ func getObjectMeta(controllerResource *piraeusv1.LinstorCSIDriver, nameFmt, comp
 		Name:      fmt.Sprintf(nameFmt, controllerResource.Name),
 		Namespace: controllerResource.Namespace,
 		Labels: map[string]string{
-			"app.kubernetes.io/name":       kubeSpec.ControllerRole,
+			"app.kubernetes.io/name":       component,
```

Comment:
Why? We already have the component label for that?

Reply:
`csi-node` daemonset has following labels set:
```yaml
app.kubernetes.io/component: csi-node
app.kubernetes.io/instance: linstor
app.kubernetes.io/managed-by: piraeus-operator
app.kubernetes.io/name: csi-controller
```

I found it a bit confusing 🤔

Reply:
Oh, sure. should probably be `kubeSpec.CSINodeRole`...

Reply:
Yeah, it is passed as third argument to the function:
https://github.com/piraeusdatastore/piraeus-operator/blob/51a74d33b21eba2b262eea173c0b58dc691dfb06/pkg/controller/linstorcsidriver/linstorcsidriver_controller.go#L305

Reply:
:bulb: Ah, makes sense. Thinking some more, shouldn't the name just be "linstor-csi" (or piraeus-csi...) :thinking: 

We already have the component label for the distinction between controller and node.